### ISSUE TITLE
[blog] End of year recap series

### ISSUE THREAD

**Issue by choldgraf** [2i2c-org/team-compass#333]
### Description

2i2c has been in existence for _roughly_ a year now! 🎉 

In that time, we've done quite a lot. We should write a short recap post that describes what we've done, and what we'd like to do next.

### Value / benefit

For 2i2c's stakeholders, this is an opportunity to demonstrate what we've been up to, show what an impressive bunch we are, and give an indication for what to expect in the future.

### Implementation details

There are a few main stakeholders that I think would appreciate this:

1. Organizations that might want to work with 2i2c for their hub
2. Organizations that care about open source support
3. Organizations that are interested in 2i2c from an organizational standpoint
4. Ourselves!

I think we should tailor some kind of content for each of these stakeholders.

I know this is a last-second addition, but I'd like to finish this up before the end of the year, we'll see how that goes :grim

### Tasks to complete

Here's my plan:

Split this into 3 blog posts, I'll try to have a draft of the first today, to post tomorrow morning, then Thurs, then next Mon.
Each post will focus on a different major area of 2i2c. I'll use the list below to keep track of each:

- [x] https://github.com/2i2c-org/2i2c-org.github.io/pull/90
- [x] Infrastructure
	- https://2i2c.org/blog/2022/ci-cd-improvements/
- [x] Organization
	- https://2i2c.org/blog/2022/q1-update/
	- https://2i2c.org/blog/2022/job-product-community-lead/

**Comment by choldgraf** [comment]
### Update: will de-prioritize this for a bit

In our meeting today we agreed that this is an important task, but it's not extremely important to do this as quickly as possible. I'm going to remove myself as "assigned" on this one, in order to free up my time to focus on other things, but will return to this in a "backburner" sense if I have bandwidth.

To make sure that this doesn't just stay open indefinitely, if we haven't made any progress on it by, say, July 2022, then we should just close this issue.

**Comment by choldgraf** [comment]
My feeling is that we can just close this one and mark it as done - we've released a few more blog posts since the start of the year, some touching on infrastructure and some on organizational stuff, and I think these are more impactful than if we did a more generic "summary" post from last year. So I'll close this and consider it finished, and we can continue releasing more blog posts about our progress in the coming weeks.

If others disagree I'm happy to discuss coming up with a summary post and re-opening!
